### DISCUSSION TITLE
Poll-based workflow execution

### DISCUSSION THREAD

**Discussion by RafaelPalomar** [argoproj/argo-workflows#7032] [Q&A] [answered]
Hello. Is there any way to execute a pipeline doing a poll-based detection of changes in a repository? In my setup I'm not able to rely on webhooks and argo-events. Thanks in adavance.

**Comment by sarabala1979** [comment, marked as answer]
You can try with cronworkflow that periodically trigger workflow which has http template with GET request based the response you can continue workflow

**Reply by RafaelPalomar** [reply]
That seems a reasonable solution. Thank you!
